essential_reading/ParallelProcessing/Parallel_Clusters.md

```diff
@@ -0,0 +1,5 @@
+# Parallel / Clusters
```

Comment:
Does the ARC HPC group have instructional materials on parallel processing? I am on the fence about this material...my gut reaction was "there are probably other sources that explain this stuff that we can link to"...but then I thought maybe it would be valuable to demonstrate how to do parallel computing on our HPCs. But THEN I though - the ARC HPC group should definitely have training materials on this! Just wondering if you've explored what's already out there or if we need to look around and see if this is something that we can just point to someone else to provide.

Reply:
I'd like to have a short curated list of such resources.  so then the various other notebooks scattered throughout this repo could include something like this in a markdown cell: 

```
:::{sidebar}
For a primer on parallelism, see [here](path/to/Parallel_Clusters.md)
:::
```

The `{sidebar}` text block doesn't render fancy in straight markdown, but it *WILL* make a sidebar when the notebook is rendered for JupyterBook.